src/App.js

```diff
@@ -18,6 +18,7 @@ const App = () => {
       <BrowserRouter>
         <Navbar />
         <Routes>
+          <Route path="/" element={<HomePage />} />
```

Comment:
This is okay but a better solution is to redirect to HomePage in case the path is /